src/Cardiompp.WebApi/Controllers/v1/DoctorController.cs

```diff
@@ -34,5 +35,22 @@ public async Task<IActionResult> GetByCrm(string crm)
 
             return Ok(response);
         }
+
+        /// <summary>
+        /// Get merchant by email and password.
+        /// </summary>
+        /// <param name="loginRequest">Login request.</param>
+        /// <returns></returns>
+        [HttpPost]
+        [Route("login")]
+        public async Task<IActionResult> Login([FromBody] GetDoctorByEmailAndPasswordRequest loginRequest)
+        {
+            var response = await DoctorService.GetByEmailAndPassword(loginRequest);
```

Comment:
Apenas para compartilhar ideia. 

Acho um pouco estranho a entidade Doctor ter email e senha. Eu teria criado uma entidade User, e ela teria papéis no sistema, como a de Doutor. Esses papéis seriam apenas uma definição de permissionamento. Até poderia ter uma relação entre Doctor e o User para ligar as coisas, mas seriam entidades separadas. 

Se posteriormente você quiser ter outros tipos de usuários no sistema, você atrelou todo o esquema de autenticação a uma entidade específica, ao invés de uma genérica que pode ser relacionar com qualquer outra. Por exemplo, se você quisesse ter um usuário de administração do sistema, para gerenciar algum dado de domínio e etc, esse teria que ser um doutor, o que não faz muito sentido. Ou teria que fazer outra implementação da autenticação, o que também não é bom.

Reply:
Entendi, vou estudar sobre!